src/Command/BakeSimpleMigrationCommand.php

```diff
@@ -188,6 +188,10 @@ public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionPar
             'short' => 'f',
             'boolean' => true,
             'help' => 'Force overwriting existing file if a migration already exists with the same name.',
+        ])->addOption('source', [
+            'short' => 's',
+            'default' => 'Migrations',
+            'help' => 'Name of the folder in which the migration should be saved',
```

Comment:
Does it name the default? Otherwise we should clarify that in the help text.

Reply:
The default value is added at the end of the help text. In this case it would be 

> Name of the folder in which the migration should be saved. (default: Migrations)